music21/musicxml/m21ToXml.py

```diff
@@ -488,12 +488,26 @@ def fromMeasure(self, m):
         representation of a Measure, not for partial
         solutions in Part or Stream production.
         '''
+        m.coreGatherMissingSpanners()
```

Comment:
I wonder if this should happen by default in Stream.makeNotation()?  or will it lead to redundancies?